test/unit/http_proxy_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'test_helper'
+
+class HttpProxiesTest < ActiveSupport::TestCase
+  let(:http_proxy) {FactoryGirl.create(:http_proxy)}
+
+  test 'create' do
+    proxy = HttpProxy.new(:name => 'foobar', :url => "http://someurl:5000")
+
+    assert proxy.save
+  end
+
+  test 'create requires url' do
```

Comment:
This can be replaced by shoulda-matchers to reduce the boilerplate.
```ruby
class HttpProxiesTest < ActiveSupport::TestCase
  should validate_presence_of(:url)
end
```